aldryn_config.py

```diff
@@ -93,6 +111,12 @@ def to_settings(self, data, settings):
 
         settings['CMS_PERMISSION'] = data['permissions_enabled']
 
+        if data['cms_content_cache_duration']:
+            settings['CMS_CACHE_DURATIONS']['content'] = data['cms_content_cache_duration']
```

Comment:
yes it's defined through django CMS @GaretJax 
http://docs.django-cms.org/en/release-3.3.x/reference/configuration.html#cms-cache-durations